launchbar/org.eclipse.launchbar.ui/src/org/eclipse/launchbar/ui/internal/dialogs/NewLaunchConfigEditPage.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2014 QNX Software Systems and others.
+ * Copyright (c) 2014, 2024 Renesas Electronics Europe and others.
```

Comment:
Please don't change copyright ownership on files :-) The proper copyright holder listed should be the original one, with "and others" covering Renesas. (Same applies to all copyright blocks)